distribution/ddf-common/src/main/resources-filtered/etc/system.properties

```diff
@@ -81,23 +81,20 @@ org.codice.ddf.system.registry-id=
 #
 # Uncomment the properties based on the desired configuration:
 
-# For Solr External Provider:
+# For Single Solr server:
 solr.client=HttpSolrClient
 solr.http.port=8994
 solr.http.url=http://localhost:${solr.http.port}/solr
-solr.data.dir=_DO_NOT_EXPAND_${ddf.data}/solr
 solr.managed.internally=true
+# TODO: Is this property still meaningful?
```

Comment:
Do we want to merge in TODO's?

Reply:
Good catch Austin. We do not want to leave a "TODO" in the file.